doc/source/user_guide/meshing_workflow/meshing_workflows.rst

```diff
@@ -572,6 +572,220 @@ Switch to solution mode
 
     solver = meshing.switch_to_solver()
 
+
+2D meshing workflow
+-------------------
+Use the **2D** meshing workflow to mesh specific two dimensional geometries.
+The following example shows you how to use the 2D Meshing workflow.
```

Comment:
```suggestion
Use the **2D** meshing workflow to mesh specific two-dimensional geometries.
The following example shows how to use the 2D Meshing workflow.
```
I don't think we need to bold 2D as the title already ponts out the topic is about 2D.

Reply:
@PipKat, the bold was done to be consistent with other workflow types.